### DISCUSSION TITLE
Trigger Alarm

### DISCUSSION THREAD

**Discussion by EmileSpecs** [rainepretorius/olarm-ha-integration#14] [Q&A] [answered]
Hi @rainepretorius 

If this is possible, I'm overlooking how, but is it possible to trigger the alarm using your integration?

I've got some cameras that I also want to use as sensors to be able to trigger the alarm.

Thanks for this integration!

**Comment by rainepretorius** [comment]
Hi, It depends all on how yoir alarm has been set up. You will have to do some reverse engineering from your Armed Response's radio to your alarm and see how it is connected.

**Comment by jonathankretzmer** [comment]
Perhaps there is an action for the Panic Button like in the app?

**Reply by rainepretorius** [reply, marked as answer]
Hi, There is currently no method to trigger the panic via the Olarm API, and thus the integration. The only way now is to trigger a PGM or utility key connected to your armed response radio.
